lib/lita/handlers/reviewme.rb

```diff
@@ -118,22 +112,16 @@ def mention_reviewer(response, room: get_room(response))
 
       private
 
-      def next_reviewer(room)
-        ns_redis(room.id).rpoplpush(REDIS_LIST, REDIS_LIST)
-      end
+      def next_reviewer(room, owner = nil)
+        return unless (reviewer = ns_redis(room.id).rpoplpush(REDIS_LIST, REDIS_LIST))
 
-      def github_comment(reviewer, pull_request)
-        msg = config.github_comment_template
+        if reviewer == owner
+          return if ns_redis(room.id).llen(REDIS_LIST) == 1
```

Comment:
Bummer we have to open the escape hatch again this far down in the method. I have feels like we might be able to extract some things into other methods to clarify the story here.